src/v2/Apps/Conversation/Components/TimeSince.tsx

```diff
@@ -35,9 +35,38 @@ const exactDate = (time: string) => {
   }
 }
 
+const minutesSinceDate = (time: string | DateTime): number => {
+  if (!time) {
+    return null
+  }
+  const date = typeof time === "string" ? DateTime.fromISO(time) : time
+  return Math.floor(Math.abs(date.diffNow("minutes").toObject().minutes))
```

Comment:
#minor is that `toObject()` necessary? seems luxon `Duration` [has](https://moment.github.io/luxon/docs/class/src/duration.js~Duration.html#instance-get-minutes) minutes getter

Reply:
removed!